templates/demo/template-primary-secondary.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../components/demo/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../components/demo/demo-page.js';
+			import '../template-primary-secondary.js';
+		</script>
+	</head>
+	<body>
+		<d2l-demo-page page-title="Primary Secondary Template">
+
+			<h2>Simple</h2>
+			<d2l-demo-snippet style="max-width:100%; width: 100%;">
+				<template>
+					<d2l-template-primary-secondary>
+						<div slot="header">I'm in the HEADER slot!</div>
+						<div slot="primary">
+							I'm in the PRIMARY slot!<br /><br />Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque euismod pellentesque dui, ac porta sapien laoreet et. Suspendisse tempor purus nec arcu finibus aliquam. Quisque eu erat blandit, luctus nulla nec, tempus velit. Morbi malesuada eu mauris volutpat feugiat. Duis eget hendrerit ex. Fusce vitae sem pretium, gravida velit eget, dapibus odio. Cras a maximus risus, ut sagittis augue.<br /><br />This is the end of the PRIMARY slot!
+						</div>
+						<div slot="secondary">
+							I'm in the SECONDARY slot!<br /><br />Duis efficitur rutrum ipsum rhoncus tristique. Pellentesque nulla elit, porta in auctor id, interdum eu dui. Suspendisse turpis urna, malesuada vel luctus quis, scelerisque sagittis justo. Quisque ac nulla nec dui varius posuere in nec est.<br /><br />This is the end of the SECONDARY slot!
+						</div>
+						<div slot="footer">I'm in the FOOTER slot!</div>
+					</d2l-template-primary-secondary>
+					<style>
+						d2l-template-primary-secondary div[slot=header] {
+							background-color: #888;

```

Comment:
I'd probably do this with just inline styles, i.e. `<div slot="primary" style="background-color: #888">`.